### PR TITLE
Add a link to the shared view to the view sharing dialog #LMR-1202

### PR DESCRIPTION
**Pull request by kubedo8**

### CODE REVIEW

File: src/app/dialog/dialog.module.ts [hunk 1]
```diff
@@ -56,6 +58,8 @@ import {UserRolesPipe} from './share-view/user-roles.pipe';
     DialogBaseComponent,
     ShareViewDialogComponent,
     ShareUserComponent,
+    ShareViewInputComponent,
+    ShareViewCopyComponent,
```
Comment (livthomas) [inline src/app/dialog/dialog.module.ts]: What about creating `ShareViewDialogModule`?

File: src/app/dialog/share-view/share-view-dialog.component.ts [hunk 2]
```diff
@@ -65,12 +72,20 @@ export class ShareViewDialogComponent implements OnInit, OnDestroy {
...
 
   public ngOnDestroy() {
     this.subscriptions.unsubscribe();
   }
 
+  public copyToClipboard() {
+    document.addEventListener('copy', this.handler);
+    setTimeout(() => {
+      document.execCommand('copy');
+    });
```
Comment (livthomas) [inline src/app/dialog/share-view/share-view-dialog.component.ts]: What about creating `ClipboardService` with a single `copy()` method we can reuse throughout the app and change the implementation later if we decide to use [Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API)?

Comment (kubedo8) [reply src/app/dialog/share-view/share-view-dialog.component.ts]: But current solution use handler which depends on public property from component. So are you sure to extract this code to `ClipboardService` ?

Comment (kubedo8) [reply src/app/dialog/share-view/share-view-dialog.component.ts]: Ok I found solution.

**Comment by kubedo8** [comment]
@livthomas done ✅

**Pull request merged**